pkg/descheduler/strategies/pod_lifetime_test.go

```diff
@@ -83,6 +83,9 @@ func TestPodLifeTime(t *testing.T) {
 	p5.ObjectMeta.OwnerReferences = ownerRef4
 	p6.ObjectMeta.OwnerReferences = ownerRef4
 
+	var maxLifeTime *uint
+	maxLifeTime = new(uint)
+	*maxLifeTime = 600
```

Comment:
```
var maxLifeTime uint = 600
...
MaxPodLifeTimeSeconds: &maxLifeTime
```

will do as well.

Reply:
Fixed in 51eed624bf7370917428581acb3f4d62d3b114ad